src/scrapers/CourseScraper.js

```diff
@@ -0,0 +1,90 @@
+import * as cheerio from'cheerio';
+
+
+//getCourse('https://tvarkarasciai.vu.lt/mif/ajax_fullcalendar_events/informatika-1k-1gr-2024/group/265/?start=2025-01-06T00%3A00%3A00%2B02%3A00&end=2025-01-11T00%3A00%3A00%2B02%3A00');
+
+let Course=[];
+async function getCourse(link) {
+  const response=await fetch(link)
+  const data=await response.json();
+  data.events.forEach((event,index) =>{
+
+  const start=event.start
+  const end=event.end
+
+  // Load the title HTML into Cheerio
+  const $ = cheerio.load(event.title);
+  
+  // Select the <a> element and get the data-title attribute
+  let dataTitle = $($('a').attr('data-title')).find('a').text();
+  if(!dataTitle)dataTitle=event.title;
+
+  let dataType = $($('a').attr('data-showed_type')).text();
+  dataType=dataType.replace('Tipas: ', '');
+  dataType=dataType.replace(', ', ' ');
+  dataType=dataType.replace('ir ', '');
+  dataType=dataType.split(" ")
+  
+  let dataGroups = $($('a').attr('data-groups')).find('a').text();
+  dataGroups=dataGroups.split("gr.");
+  dataGroups=dataGroups.slice(0,-1);
+
+  let dataAcademics = $($('a').attr('data-academics')).find('a').text();
+  
+  let dataRooms= $($('a').attr('data-rooms')).find('a').text();
+  dataRooms=dataRooms.replace('(MIF-Šalt.)', 'Šaltinis');
+  dataRooms=dataRooms.replace('(MIF-Naug.)', 'Naugardukas');
+  dataRooms=dataRooms.replace('(MIF-Didl.)', 'Didlaukis');
+  dataRooms=dataRooms.replace('S-', '');
+  let Place=dataRooms.split(" ")
+  let Room=Place[0];
+  let location=Place[1]
+
+
+  let dataComments = $($('a').attr('data-comments')).text();
+  dataComments=dataComments.replace('Komentarai: ', '');
+  
+  let dataSubgroup =  $($('a').attr('data-subgroups')).text();
+  dataSubgroup=dataSubgroup.replace('Pogrupiai: ', '');
+
+
+  Course[index]={};
+
+  Course[index].start=start;
+  Course[index].end=end;
+
+  if(dataTitle){
+    Course[index].title = dataTitle
+  }
+
+  if(dataType){
+    Course[index].type = dataType
+  }
+
+  if(dataGroups){
+    Course[index].groups = dataGroups.map((e)=>parseInt(e))
+  }
+
+  if(dataSubgroup){
+    Course[index].subGroup = parseInt(dataSubgroup)
+  }  
+
+  if(dataAcademics){
+    Course[index].academic = dataAcademics
+  } 
+  
+  if(Room){
+    Course[index].room = parseInt(Room)
+  }         
+
+  if(location){
+    Course[index].location = location
+  }      
+
+  if(dataComments){
+    Course[index].notes = dataComments
+  } 
+
+  })
+}  
+export default Course
```

Comment:
- export not the actual parsed data but the function that parses it
- also rename the function to be parseLecture